src/uwtools/config/templater.py

```diff
@@ -0,0 +1,111 @@
+"""
+Support for rendering Jinja2 templates.
```

Comment:
It might make sense to revisit the merge of the templater and J2Templates. We've had a rocky past with templates, so I think we could revisit that now.

Reply:
I'll create a "Consider merging..." Jira ticket.